infrastructure/terraform/per_workspace/modules/api_storage/dynamodb.tf

```diff
@@ -15,8 +15,13 @@ module "dynamodb_table" {
 
   point_in_time_recovery_enabled = true
 
-  tags = {
-    Name = var.name
-  }
+  tags = merge(
+    {
+      Name = var.name
+    },
+    var.environment == "dev" ? {
```

Comment:
Might be worth putting comments everywhere we want to change this value of 'dev' to 'prod' with something you can search the whole repo with so you know what to change when the QA is done